src/codeu/chat/client/core/Context.java

```diff
@@ -56,6 +57,8 @@ public UserContext create(String name) {
     return users;
   }
   
+  //public ServerVersion getVersion() {
```

Comment:
Please address this. Generally, we don't need to comment out code; we have revision  history to restore from.